doc/source/whatsnew/v2.1.0.rst

```diff
@@ -25,6 +25,7 @@ Copy-on-Write improvements
 - The :class:`DataFrame` constructor, when constructing a DataFrame from a dictionary
   of Index objects and specifying ``copy=False``, will now use a lazy copy
   of those Index objects for the columns of the DataFrame (:issue:`52947`)
+- :meth:`DataFrame.head` and :meth:`DataFrame.tail` will now return copies (:issue:`54011`)
```

Comment:
Should we clarify this as "deep copy"? With CoW, the methods already do return an object that behaves as a copy / is a shallow copy.

Reply:
Makes sense, thx